src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1123,25 +1124,38 @@ void DynamoView_KeyDown(object sender, KeyEventArgs e)
 
             var vm = dynamoViewModel.BackgroundPreviewViewModel;
 
-            if (e.IsRepeat)
+            if (!sw.IsRunning && !vm.NavigationKeyIsDown)
```

Comment:
Let's add a comment here explaining why this stopwatch exists with a reference to this bug.